src/pages/wallet/add.custom.token.confirm.js

```diff
@@ -137,6 +138,14 @@ class AddCustomToken extends Component {
         });
       } catch (error) {
         const notification = getErrorNotification(error.code) || getDefaultErrorNotification();
+
+        if (!getErrorNotification(error.code)) {
```

Comment:
```suggestion
		const decodedNotification = getErrorNotification(error.code);
        const notification = decodedNotification || getDefaultErrorNotification();

        if (!decodedNotification) {
```

Reply:
Great! I can make this change in the places it is located. My initial thought was not to touch any of the existing code, but this may make the code quicker by only having to call `getErrorNotifi` once